### PR TITLE
Added https connection and SSL verification

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25738

## Description
Added a configuration checkbox to use HTTPS and not HTTP
Added the 'trust any cert...' option

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 1]
```diff
@@ -19,6 +19,16 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:24]:
```suggestion
```

File: Packs/ipinfo/Integrations/integration-Ipinfo.yml [hunk 2]
```diff
@@ -19,6 +19,16 @@ configuration:
...
+  type: 8
+  required: false
+- display: Use HTTPS connections
+  name: use_https
+  defaultvalue: ""
```
Comment (Itay4) [inline Packs/ipinfo/Integrations/integration-Ipinfo.yml:29]:
```suggestion
```

Comment (ShahafBenYakir) [reply Packs/ipinfo/Integrations/integration-Ipinfo.yml:29]: lama lo tsarich?

Comment (Itay4) [reply Packs/ipinfo/Integrations/integration-Ipinfo.yml:29]: just unnecessary

**Comment by kirbles19** [comment]
@ShahafBenYakir Done.

**Comment by ShahafBenYakir** [comment]
Changes are not related to MP failure. Force merging.

**Pull request merged**